test/modules/obsagent-client/src/otel_client.rs

```diff
@@ -51,7 +51,7 @@ fn init_meter(period: f64, otlp_endpoint: String) -> metrics::Result<PushControl
 
 pub async fn run(config: Config) -> Result<(), OTelClientError> {
     let _started = init_meter(
-        1.0 / config.otel_config.push_rate,
+        config.otel_config.push_period,
```

Comment:
Seems unlikely, but if `init_meter` had an overload that took in `Duration` then we could use a more appropriate type in the config. Still fine to me in the current state, especially because it is a diagnostic tool for benchmarking.

Reply:
Good point! Actually, `init_meter` is a method in our otel_client module so I was able to change it to accept a `Duration` instead of an `f64`. Also updated the config structs to use `Duration`.